addons/l10n_au/data/localisation.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+	<data noupdate="1">
+		<record id="base.AUD" model="res.currency">
+			<field name="position">before</field>
+			<field name="active" eval="True"/>
```

Comment:
Is it activated, though?  When we did our first test installs of enterprise, the only currencies showing as active were EUR / USD.  We also want to ensure that both AUD and NZD are available.